src/lib/eDSL.ml

```diff
@@ -302,18 +301,25 @@ let depends_on l =  Depends_on l
 let on_success_activate n = On_success_activate n
 let on_failure_activate n = On_failure_activate n
 
+type done_when_option = [
+  | `Dependencies_are
+  | `Is_verified of Condition.t
+  | `Product_is 
```

Comment:
Now that I know it, it is! I also didn't see the docs until after I made the comment; I think it's probably clear enough between the two.